src/components/fx/hover.js

```diff
@@ -206,7 +206,7 @@ exports.loneHover = function loneHover(hoverItems, opts) {
     });
 
     var container3 = d3.select(opts.container);
-    var outerContainer3 = opts.outerContainer ? d3.select(opts.outerContainer) : container3;
+    var outerContainer3 = opts.outerContainer || container3;
```

Comment:
```suggestion
    var outerContainer3 = opts.outerContainer || opts.container;
```
Not sure if this has anything to do with the test problems, but to be consistent `outerContainer` should always be a node, not a d3 selection (so should probably not have the `3` in its name either 😁)